article/_posts/2021-01-14-MedAL.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: review
+title: "MedAL: Accurate and Robust Deep Active
+Learning for Medical Image Analysis"
+tags: active-learning deep-learning medical classification
+author: "Thierry Judge"
+cite:
+    authors: "Asim Smailagic, Hae Young Noh, Aure´lio Campilho, Pedro Costa, Devesh Walawalkar, Kartik Khandelwal,
+Mostafa Mirshekari, Jonathon Fagert, Adrian Galdran, Susu Xu"
+    title:   "MedAL: Accurate and Robust Deep Active
+Learning for Medical Image Analysis"
+    venue:   "ICMLA 2018"
+pdf: "https://arxiv.org/ftp/arxiv/papers/1809/1809.09287.pdf"
+---
+
+# Introduction
+
+The authors introduce a novel method for active learning that aims to select more representative samples that standard active learning heuristics. 
+
+# Summary of Contributions
+
+* Novel AL sampling method that maximizes the representativeness of samples selected for labeling.
+* Better initialization by selecting the initial pool with standard feature extraction and description methods
+
+# Existing Methods 
+
+Consider an empty training set $$D_{train}$$ and an unlabeled dataset  $$D_{oracle}$$
+
+Standard Active learning loop
+
+1. Label $n$ samples from  $$D_{oracle}$$ and add them to $$D_{train}$$ .
+2. Train model on $$D_{train}$$ .
+3. Evaluate the model's uncertainty on $$D_{oracle}$$ .
+4. Label $$n$$ most uncertain samples and add them to $$D_{train}$$ .
+5. Repeat steps 2 to 4.
+
+Many heuristics can be used to evaluate uncertainty. 
+
+# Method
+
+## Sampling method
+
+In this work, the authors use rich data descriptions $$f(x) \in R^n$$ to select the most representative sample. This is done by evaluating the average distance with all the samples in $$D_{train}$$. 
```

Comment:
Should it be "representative samples" here, as in you have a ranking of the samples, and not only the very best one?